DTAConfig/OptionPanels/DisplayOptionsPanel.cs

```diff
@@ -1,15 +1,13 @@
+using Localization;
 using ClientCore;
 using ClientGUI;
-using Localization;
-using Microsoft.Win32;
 using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
 using Rampastring.Tools;
 using Rampastring.XNAUI;
 using Rampastring.XNAUI.XNAControls;
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
```

Comment:
What is the point of removing `Microsoft.Win32` and `System.Diagnostics` from usings and repeating the namespaces several times in the code?

Reply:
The Microsoft.Win32 namespace for example is only used in code when the TS constant is defined. Because of this, depending on your IDE settings etc., the namespace may be marked as not needed when the constant is not defined. You may accidently or willfully remove the using and build your code without problem when working on code that does not require TS. By removing the usings and appending them directly to the types inside the TS code it makes it much harder for developers to accidently break builds other than the one they're working on.

Reply:
Can't you do this instead?

```cs
#if TS
using Microsoft.Win32;
using System.Diagnostics;
#endif
```

Reply:
Updated